src/auth/index.js

```diff
@@ -169,17 +169,147 @@ export default class Auth {
     const payload = apply(
       {
         parameters: {
-          refreshToken: { required: true, toName: 'refresh_token' },
-          scope: { required: false }
-        }
+          refreshToken: {required: true, toName: 'refresh_token'},
+          scope: {required: false},
+        },
       },
-      parameters
+      parameters,
     );
     return this.client
       .post('/oauth/token', {
         ...payload,
         client_id: this.clientId,
-        grant_type: 'refresh_token'
+        grant_type: 'refresh_token',
+      })
+      .then(responseHandler);
+  }
+
+  /**
+   * Starts the Passworldess flow with an email connection
+   *
+   * @param {Object} parameters passwordless parameters
+   * @param {String} parameters.email the email to send the link/code to
+   * @param {String} parameters.send the passwordless strategy, either 'link' or 'code'
+   * @param {String} parameters.authParams optional parameters, used when strategy is 'linḱ'
+   * @returns {Promise}
+   *
+   * @memberof Auth
+   */
+  passwordlessWithEmail(parameters = {}) {
+    const payload = apply(
+      {
+        parameters: {
+          email: {required: true},
+          send: {required: false},
+          authParams: {required: false},
+        },
+      },
+      parameters,
+    );
+    return this.client
+      .post('/passwordless/start', {
+        ...payload,
+        connection: 'email',
+        client_id: this.clientId,
+      })
+      .then(responseHandler);
+  }
+
+  /**
+   * Starts the Passworldess flow with an SMS connection
+   *
+   * @param {Object} parameters passwordless parameters
+   * @param {String} parameters.phoneNumber the phone number to send the link/code to
+   * @param {String} parameters.send the passwordless strategy, either 'link' or 'code'
```

Comment:
Isn't `link` not allowed when using SMS? See https://auth0.com/docs/api/authentication#get-code-or-link:

> You have three options for passwordless authentication:
> 
> - Send a verification code using email.
> - Send a link using email.
> - Send a verification code using SMS.

Reply:
That's a good question actually. I don't know the answer. But on [this doc](https://auth0.com/docs/connections/passwordless/guides/sms-otp#configure-passwordless-sms-settings) they still mention link as a possible strategy for SMS connections

> Only the last one-time password **(or link)** issued will be accepted.

Let me ask the right team and come back

Reply:
Remove this param from the doc-block.